lsan_suppress.txt

```diff
@@ -0,0 +1,15 @@
+leak:obmalloc.c
+leak:stringio.c
+leak:pyo3
+leak:crypto
+leak:unicodeobject.c
+leak:bytesobject.c
+leak:listobject.c
+leak:gcmodule.c
+leak:PyThread
+leak:arrow
+leak:numpy
+leak:QtCore
+leak:libxml2
+leak:libxslt
+leak:kh_init_pymap
```

Comment:
Was having a hard time reproducing this but LSAN is convinced that we have the following leak:

```sh
Direct leak of 320 byte(s) in 8 object(s) allocated from:
    #0 0x74b912295749 in __interceptor_calloc ../../../../libsanitizer/asan/asan_malloc_linux.cpp:77
    #1 0x74b8e9062bae in traced_calloc ../pandas/_libs/include/pandas/vendored/klib/khash_python.h:37
    #2 0x74b8e90790a9 in kh_init_pymap ../pandas/_libs/include/pandas/vendored/klib/khash_python.h:356
    #3 0x74b8e9255ca5 in __pyx_pf_6pandas_5_libs_9hashtable_17PyObjectHashTable___cinit__ pandas/_libs/hashtable.cpython-310-x86_64-linux-gnu.so.p/pandas/_libs/hashtable.pyx.c:130190
    #4 0x74b8e9255b67 in __pyx_pw_6pandas_5_libs_9hashtable_17PyObjectHashTable_1__cinit__ pandas/_libs/hashtable.cpython-310-x86_64-linux-gnu.so.p/pandas/_libs/hashtable.pyx.c:130155
    #5 0x74b8e93e5df3 in __pyx_tp_new_6pandas_5_libs_9hashtable_PyObjectHashTable pandas/_libs/hashtable.cpython-310-x86_64-linux-gnu.so.p/pandas/_libs/hashtable.pyx.c:188974
    #6 0x5dfd812b4908 in type_call /usr/local/src/conda/python-3.10.13/Objects/typeobject.c:1123
    #7 0x5dfd812b4908 in _PyObject_MakeTpCall /usr/local/src/conda/python-3.10.13/Objects/call.c:215
```

I wasn't able to reproduce this using the PyObjectHashTable class directly; only when running the IO JSON tests via pytest was I seeing it.